src/ap/ieee802_1x.c

```diff
@@ -1905,7 +1905,8 @@ struct radius_vendor_specific_info {
 
 #define RADIUS_VENDOR_ID_ARUBA (14823)
 enum {
-	RADIUS_VENDOR_ATTR_ARUBA_USER_GROUP = 36
+	RADIUS_VENDOR_ATTR_ARUBA_WIRED_USER_GROUP = 36,
```

Comment:
@parthafff add the extra space before `__ignore__` in `sta_info.c`

Reply:
Done. I have kept it consistent and added the space where we add user name.